HACKING.md

```diff
@@ -54,7 +54,7 @@ targets without running any command, just by looking at the source
 tree. This is used to automatically generate the `<package>.opam`
 files for all Jane Street packages.
 
-To implement this, the build rules are described using a build arrow,
+To implement this, the build rules are described using a build functor,
```

Comment:
By the way, the word "functor" in the context of OCaml pretty much always means OCaml functor by default, not category-theoretical functor. I think people are using various language tricks to work around that ambiguity, such as "type constructor", "type", "applicative" etc. I sometimes (rarely) just go and clarify: "(category-theoretical) functor"  :-)

Reply:
Good point! I also was a bit hesitant to use "build functor".

Perhaps, we could use "build description" or "build specification" in the docs? Happy to make the change once we agree on the term.

Reply:
I vote for "build description"

Reply:
Switched to "build description".